index.js

```diff
@@ -0,0 +1,76 @@
+function sayHello(name) {
+    console.log("Hello, " + name + "!");{
```

Comment:
add check for `name` type to avoid unpredictable results. what will be if I call your function with object for example?